### PR TITLE
Add gameup server

### PR DESCRIPTION
**Pull request by MatinHQ**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

## Further comments

play.voidpixel.ir is also linked to us

### CODE REVIEW

File: minecraft_servers/gameup/background.png
Comment (maaaathis) [inline minecraft_servers/gameup/background.png:1]: Compress this file

File: minecraft_servers/gameup/background@2x.png
Comment (maaaathis) [inline minecraft_servers/gameup/background@2x.png:1]: Compress this file

File: minecraft_servers/gameup/manifest.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+  "server_name": "gameup",
+  "nice_name": "§8§l[§c§lGame§f§lUP§8§l]§8",
+  "direct_ip": "mc.gameup.ir",
+  "server_wildcards": [
+    "%.gameup.ir",
```
Comment (maaaathis) [inline minecraft_servers/gameup/manifest.json:6]: Doesn't look like a real wildcard:
image

File: minecraft_servers/gameup/manifest.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "%.voidpixel.ir"
```
Comment (maaaathis) [inline minecraft_servers/gameup/manifest.json:7]: Doesn't look like a real wildcard:
image

File: minecraft_servers/gameup/manifest.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{
+  "server_name": "gameup",
+  "nice_name": "§8§l[§c§lGame§f§lUP§8§l]§8",
```
Comment (maaaathis) [inline minecraft_servers/gameup/manifest.json:3]: I'm not sure right now if colorcodes are okay in the name...

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/gameup/manifest.json (hunk 3 above)
Comment (MatinHQ) [reply minecraft_servers/gameup/manifest.json:3]: Also we can test that too

**Comment by MatinHQ** [comment]
@maaaathis Optimized images also corrected wild cards

**Pull request closed without merging**